lighthouse-core/gather/driver.js

```diff
@@ -666,7 +665,7 @@ class Driver {
 
     /** @type {NodeJS.Timer|undefined} */
     let lastTimeout;
-    let cancelled = false;
+    let canceled = false;
```

Comment:
boooo to one L :P

Reply:
I prefer Mr. Webster's simplified spellings, but I could care more.